docs/developer-docs/latest/developer-resources/content-api/integrations/11ty.md

```diff
@@ -1,3 +1,8 @@
+---
+title: Get started with 11ty - Strapi Developer Documentation
+description: Build powerful applications using Strapi, the leading open-source headless cms and 11ty.
```

Comment:
Just wondering why 'cms' is not uppercase (→ CMS). Is it because search engines index words as lowercase anyway?

```suggestion
description: Build powerful applications using Strapi, the leading open-source headless CMS and 11ty.
```